assignment-01.md

```diff
@@ -5,26 +5,56 @@ E.g. Question <br>
 
 1. What is the difference between private limited and public limited company? State one example for each type.
 
+   A public limited company is a company listed on a recognized stock exchange and the stocks are traded publicly. A private limited company is neither listed on the stock exchange nor are they traded. Public limited: Coca Cola. Private limited: Nike.
+   https://legaldocs.co.in/blog/private-limited-company-vs-public-limited-company
+
 2. Why is Berjaya Sports Toto Berhad registered under Non-Shariah Compliant?
 
+   Islamic finance transaction is challenged based on Shariah non complaint. 
+   https://www.mdpi.com/1911-8074/13/1/2/htm
+
 3. List down 3 Non-Shariah Compliant Business Activities. 
 
+   Robbery, Gambling, Corruption. 
+   https://www.malaysia.gov.my/portal/content/27678
+
 4. Name the stock exchange of Malaysia.
 
+   Bursa Malaysia. 
+   https://www.bursamalaysia.com/
+
 5. Create a new folder name 'AR' in your _repo main branch_. Upload Annual Report 2018 - 2020 for companies below: <br>
 (a) ViTrox Corp <br>
 (b) Pentamaster Corp <br> 
 *There should be at least 6 PDF files. <br>
+[vitrox_ar2018.pdf](https://github.com/minor-management/assignment-01-Pzzzi/files/8309232/vitrox_ar2018.pdf)
+[vitrox_ar2019.pdf](https://github.com/minor-management/assignment-01-Pzzzi/files/8309234/vitrox_ar2019.pdf)
+[vitrox_ar2020.pdf](https://github.com/minor-management/assignment-01-Pzzzi/files/8309235/vitrox_ar2020.pdf)
+https://www.vitrox.com/investor/annual-report.php
+[ar2018.pdf](https://github.com/minor-management/assignment-01-Pzzzi/files/8309238/ar2018.pdf)
+[ar2019.pdf](https://github.com/minor-management/assignment-01-Pzzzi/files/8309239/ar2019.pdf)
+[ar2020.pdf](https://github.com/minor-management/assignment-01-Pzzzi/files/8309241/ar2020.pdf)
```

Comment:
Referring to the instruction above, you are required to **create a new folder in your main branch**. And upload all PDF files there.